wagtail/images/blocks.py

```diff
@@ -148,7 +148,7 @@ def to_python(self, value):
     def bulk_to_python(self, values):
         values = list(values)
 
-        if not values:
+        if not values or values[0] is None:
```

Comment:
I don't think this is correct - the result of `bulk_to_python` should be a list of the same length as its input. A list such as `[None, 123]` should return `[None, <Image: id=123>]`.